docs/learn/benefits-lukso-standards.md

```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
 - A music dApp can only update a list of music playlists in the Universal Profile's storage.
 - A family member can be granted recovery access for trusted third-party recovery.
 
-### Features of LSP token standards over ERC20 and ERC721
+## LSP7/8 Token standards vs. ERC20/721
+
+:::success Useful guides
+
+Interested to migrate your token or NFT collection? See our hands-on developer guides:
+
+- [Migrate from ERC20 to LSP7](./migrate/migrate-erc20-to-lsp7.md)
+- [Migrate from ERC721 to LSP8](./migrate/migrate-erc721-to-lsp8.md)
+
+:::
+
+Below are the benefits offered by the LUKSO Token standards [**LSP7 Digital Asset**](../standards/tokens/LSP7-Digital-Asset.md) and [**LSP8 Identifiable Digital Asset**](../standards/tokens/LSP8-Identifiable-Digital-Asset.md).
+
+| Feature                                                               | Benefits                                                                                                                                                                                                                                                                                                                            |
+| --------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **🔘 Easier functions for developers**                                | You are a developer? Stop learning different functions for each new ERC token standard! (ERC20, ERC721, ERC1155, etc...).<br/>LSP7 and LSP8 use the **same function names** for transfer and operator approvals, whether it is a token or NFT transfer.                                                                             |
+| **🗄️ Unlimited & Dynamic Metadata for Tokens & NFTs**                 | Your NFT is not only an image. It can now hold as many information as imagined (custom traits, attributes, ...). This information can evolve overtime.                                                                                                                                                                              |
```

Comment:
```suggestion
| **🗄️ Unlimited & Dynamic Metadata for Tokens & NFTs**                 | Your NFT is not only an image. It can now hold as many information as can be imagined (custom traits, attributes, ...). This information can evolve overtime.                                                                                                                                                                              |
```